UPGRADING.md

```diff
@@ -1,5 +1,39 @@
 # Upgrade guidelines
 
+## 0.64.0 to 0.66.0
+Before this change, [k8s events receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/k8seventsreceiver) was used to collect Kubernetes Events.
+Now we utilize [k8s objects receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/k8sobjectsreceiver) that can pull or watch any object from Kubernetes API server.
+Therefore, `clusterReceiver.eventsEnabled` is now deprecated, and to maintain the same behavior you should set
```

Comment:
https://stackoverflow.com/a/9208164

It appears these changes remove `clusterReceiver.eventsEnabled` functionality in total, making it obsolete. Is this correct?

Reply:
Yes, right, k8s object receiver is not providing exactly the same functionality as k8s events receiver. Updated.

Reply:
So https://github.com/signalfx/splunk-otel-collector-chart/pull/588#discussion_r1036766723 could also be expanded so that we provide an actual deprecation path by using the objects receiver w/ just the watched events for this value instead of obsoleting in one pass. The warning check also needs to be updated with a description of the behavior.

imo users should never have their config become "broken" during an upgrade and we should provide at least a release of warnings and ideally remediation logic in helpers (cc @dmitryax)

Reply:
Ok, so I updated it in the way that when `eventsEnabled` is set to true we return warning message and event objects are watched, so behaviour is similar (but not the same, as previous event formatting was different, and the sourcetype was `kube:events` and now it is `kube:object<resource_id>`. It is described in details in the upgrading doc.